dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:9-slim
```

Comment:
- [ ] Can you add information to thee readme about how to run docker? Also make sure that in your instructions that you inform the users to run this on port `3000` and not `9000` since the front-end is currently looking for the back-end on `3000`.